projectOne/src/oneFive.java

```diff
@@ -0,0 +1,85 @@
+                    System.out.println("Буква угадана!");
+                    for (int i = 0; i < nameArray.length; i++) {
+                        if (nameArray[i].equals(input)) {
+                            nameArrayEmpty[i] = input;
+                            if (!Arrays.asList(usedLettersArray).contains(input.toLowerCase())) {
+                                usedLettersArray[nextIndex] = input;
+                                ++nextIndex;
+                            }
+                        }
+                    }
+                    for (int i = 0; i < lastNameArray.length; i++) {
+                        if (lastNameArray[i].equals(input)) {
+                            lastNameEmptyArray[i] = input;
+                        }
+                    }
+                } else if (containsName) {
+                    System.out.println("Буква угадана!");
+                    for (int i = 0; i < nameArray.length; i++) {
+                        if (nameArray[i].equals(input)) {
+                            nameArrayEmpty[i] = input;
+                            if (!Arrays.asList(usedLettersArray).contains(input.toLowerCase())) {
+                                usedLettersArray[nextIndex] = input;
+                                ++nextIndex;
+                            }
+                        }
+                    }
+                } else if (containsLastName) {
+                    System.out.println("Буква угадана!");
+                    for (int i = 0; i < lastNameArray.length; i++) {
+                        if (lastNameArray[i].equals(input)) {
+                            lastNameEmptyArray[i] = input;
+                            if (!Arrays.asList(usedLettersArray).contains(input.toLowerCase())) {
+                                usedLettersArray[nextIndex] = input;
+                                ++nextIndex;
+                            }
+                        }
+                    }
+                } else {
+                    System.out.println("Вы не угадали!");
+                    if (!Arrays.asList(usedLettersArray).contains(input.toLowerCase())) {
+                        usedLettersArray[nextIndex] = input;
+                        ++nextIndex;
+                    }
+                }
+                System.out.println("Ваш результат: " + Arrays.toString(nameArrayEmpty).replace("[", "").replace("]", "").replace(",", "").replace(" ", "") + " " + Arrays.toString(lastNameEmptyArray).replace("[", "").replace("]", "").replace(",", "").replace(" ", ""));
+            } else {
+                System.out.println("Вы не угадали!");
+                System.out.println("Ваш результат: " + Arrays.toString(nameArrayEmpty).replace("[","").replace("]","").replace(",","").replace(" ","") + " " + Arrays.toString(lastNameEmptyArray).replace("[","").replace("]","").replace(",","").replace(" ",""));
+            }
+        }
+        System.out.println("Игра окончена!");
+    }
```

Comment:
A lot of duplicate code here you should create a separate method for it or reorganize your cycle